WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/TagsAndCategoriesUseCase.kt

```diff
@@ -147,6 +160,7 @@ class TagsAndCategoriesUseCase
         return ListItemWithIcon(
                 icon = R.drawable.ic_folder_multiple_white_24dp,
                 text = text,
+                percentageOfMaxValue = tag.views.toDouble() / maxViews.toDouble(),
```

Comment:
If all the views are 0, `maxViews` is also 0 and the result of 0/0 is NaN. It should be `null` instead so we hide the bars instead when this happens. I don't think this could happen with the current API responses but I'd rather be safe.

Reply:
For sure.  Will adjust it.